Library/Homebrew/test/test_pkg_version.rb

```diff
@@ -35,7 +35,8 @@ def test_to_s
     assert_equal "1.0_1", PkgVersion.new(Version.new("1.0"), 1).to_s
     assert_equal "1.0", PkgVersion.new(Version.new("1.0"), 0).to_s
     assert_equal "1.0", PkgVersion.new(Version.new("1.0"), 0).to_s
-    assert_equal "HEAD", PkgVersion.new(Version.new("HEAD"), 1).to_s
+    assert_equal "HEAD_1", PkgVersion.new(Version.new("HEAD"), 1).to_s
+    assert_equal "HEAD-ffffff_1", PkgVersion.new(Version.new("HEAD-ffffff"), 1).to_s
```

Comment:
Yes, it is. But see my other code comments and my non-code comment https://github.com/Homebrew/brew/pull/478#issuecomment-231606435 below (in case you agree with that) for where I'm coming from and why I'm suggesting `Version.create`.